plugins/dig-now.cpp

```diff
@@ -0,0 +1,870 @@
+/*
+ * Simulates completion of dig designations.
+ */
+
+#include "DataFuncs.h"
+#include "PluginManager.h"
+#include "TileTypes.h"
+#include "LuaTools.h"
+
+#include "modules/Gui.h"
+#include "modules/Maps.h"
+#include "modules/MapCache.h"
+#include "modules/Random.h"
+#include "modules/Units.h"
+#include "modules/World.h"
+
+#include <df/historical_entity.h>
+#include <df/map_block.h>
+#include <df/reaction_product_itemst.h>
+#include <df/tile_designation.h>
+#include <df/tile_occupancy.h>
+#include <df/ui.h>
+#include <df/unit.h>
+#include <df/vermin.h>
+#include <df/world.h>
+#include <df/world_site.h>
+
+DFHACK_PLUGIN("dig-now");
```

Comment:
@myk002 I think this is the only thing that would prevent me from including `dig-now.cpp` in the channel-safely plugin build